proto/hummock.proto

```diff
@@ -201,6 +201,7 @@ message CompactTask {
   map<uint32, TableOption> table_options = 17;
   uint64 current_epoch_time = 18;
   uint64 target_sub_level_id = 19;
+  uint64 input_file_size = 20;
```

Comment:
It is no helpful because we can calculate input_file_size by `input_ssts`. I do not think we must add a new field and it would cause we must check its correctness.

Reply:
Alright. My intention was to avoid calculating it repeatedly.